__mocks__/dimensions.mock.ts

```diff
@@ -1,3 +1,7 @@
+import dayjs from "dayjs";
+
+const todaysDate = dayjs().format("YYYY-MM-DD");
```

Comment:
Some of the tests were dependent on the date being november 13, 2019. I changed that.